content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -184,9 +186,97 @@ data:
     ["172.16.0.1"]
 ```
 
-## What's next
-- [Debugging DNS Resolution](/docs/tasks/administer-cluster/dns-debugging-resolution/).
-
 {{% /capture %}}
 
+# Customizing CoreDNS
+
+This section provides with details on [CoreDNS](https://coredns.io/) as a service discovery.
 
```

Comment:
it is possible to add here a link to the page Using [CoreDNS for Service Discovery](https://kubernetes.io/docs/tasks/administer-cluster/coredns/) ?